iam-test-token/README.md

```diff
@@ -25,7 +25,11 @@ jobs:
       - name: IAM token
         uses: extenda/actions/iam-test-token@v0
         with:
-          service-account-key: ${{ secrets.SECRET_AUTH }}
+          service-account-key: ${{ secrets.GCLOUD_AUTH_STAGING }}
```

Comment:
Cloud we simplify the README so it includes two examples instead of this?

I'd prefer to show one example which is the basic usage, where we utilize all the defaults. Essentially what we want all teams to use. And then also a second example which sets all options.